tools/codegen/gen_lazy_tensor.py

```diff
@@ -235,12 +240,13 @@ def gen_key(func: FunctionSchema) -> Tuple[str, str]:
             "ATen/Functions.h",
             "ATen/MetaFunctions.h",
             "ATen/Operators.h",
+            "ATen/native/CPUFallback.h",
             "torch/csrc/lazy/core/lazy_graph_executor.h",
             "torch/csrc/lazy/core/metrics.h",
             "torch/csrc/lazy/core/shape.h",
-            "lazy_tensor_core/csrc/ts_backend/aten_eager_fallback.h",
-            f"{output_dir}/{backend_key}NativeFunctions.h",
-            f"{output_dir}/{backend_key}LazyIr.h",
+            f"torch/csrc/lazy/generated/{backend_key}NativeFunctions.h",
+            "torch/csrc/lazy/generated/LazyIr.h",
```

Comment:
@wconstab Removing this `output_dir` argument effectively forces all LTC backends to use only the generated `LazyIr.h` and `LazyNativeFunctions.h` from the TS backend. This needs to be changes to accept a variable `output_dir` again and set the `output_dir` for the default case to be`torch/csrc/lazy/generated`

Reply:
Good catch,  I'm not sure why I did that, maybe there was a reason, but I will try to put it back as an argument.